modin/experimental/scripts/cluster.py

```diff
@@ -0,0 +1,146 @@
+import os
+import subprocess
+import yaml
```

Comment:
@ericl can you comment on whether whether it makes sense to support an autoscaler config that takes in a bunch of IP addresses for nodes in an existing cluster?

Reply:
Yes, that should work